datasource/src/main/scala/cromwell/pipeline/database/LiquibaseUtils.scala

```diff
@@ -15,17 +16,30 @@ object LiquibaseUtils {
   private val DefaultContexts = new Contexts()
   private val DefaultLabelExpression = new LabelExpression()
 
-  def updateSchema(changeLogResourcePath: String)(jdbcConnection: Connection): Unit = {
+  def updateSchema(
+    changeLogResourcePath: String
+  )(jdbcConnection: Connection): Try[Unit] = {
     val liquibaseConnection = new JdbcConnection(jdbcConnection)
     Try {
       val database = DatabaseFactory.getInstance.findCorrectDatabaseImplementation(liquibaseConnection)
       val liquibase = new Liquibase(changeLogResourcePath, new ClassLoaderResourceAccessor(), database)
 
       liquibase.update(DefaultContexts, DefaultLabelExpression)
     } match {
-      case Failure(exception) => log.error("An error during the schema update via liquibase.", exception)
-      case Success(_)         => log.error("Successfully updated db schema via liquibase!")
+      case Failure(exception) =>
+        liquibaseConnection.close()
+        val liquibaseUpdateSchemaException = LiquibaseUpdateSchemaException()
+        log.error(liquibaseUpdateSchemaException.message, exception)
+        Failure(liquibaseUpdateSchemaException)
+      case Success(_) =>
+        liquibaseConnection.close()
+        log.info("Successfully updated db schema via liquibase!")
+        Success(())
     }
-    liquibaseConnection.close()
   }
 }
+
+object Exception {
```

Comment:
That naming may be not very informative. 
Btw, what is the reason to wrap that exception into an object. Couldn't we just put it inside LiquiBaseUtils?

Reply:
I've seen a similar approach with other exceptions in current project, so I used it.

Reply:
We might have overlooked it accidentally.
That's not a good thing, please change this name to something more descriptive. And if you see uninformative name somewhere else - feel free to rename it too.

Reply:
Ok, I'll fix it.